### DISCUSSION TITLE
Blocking Malware via URL Blacklists

### DISCUSSION THREAD

**Discussion by Yeraze** [AdguardTeam/AdGuardHome#3646] [Q&A]
I've added a few Malware URL Blacklists to my config, and they seem to be blocking properly (e.g. malware.wicar.com gets redirected to 0.0.0.0 properly), however these are considered "Ads" from the AdGuardHome console.

Is there any way to get these properly classified as Malware, so that I can track impact & which clients are potential problems?

**Comment by timkgh** [comment]
Isn't the setting "Use AdGuard browsing security web service" good enough? Out of curiosity what lists do you use for anti-malware?
